### PR TITLE
My Gallary challenge

### PR DESCRIPTION
**Pull request by aliakhizer**
## This page has two sections. 
The first section is for Profile Details where I have used (one-dimensional)Flex to display the

1. Image
2. Title
3. Post details profile

The second section is for Gallery Photos where I have added photos in a two-dimensional layout (grid).
## desktop view
image
## mobile view
image

### CODE REVIEW

**Comment by naumanzchaudhry** [comment]
> This page has two sections. The first section is for Profile Details where I have used (one-dimensional)Flex to display the 1)Image 2)Title 3)post details profile The second section is for Gallery Photos where I have added photos in a two-dimensional layout (grid).
> 
> ## desktop view
> image
> 
> ## mobile view
> image

Text description at the top is not completely readable. Please fix this

File: my-gallary/styles.css [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  grid-area: 1/2/2/4;
+}
+
+.grid-items3 {
+  grid-area: 2/1/3/3;
+}
+.grid-items4 {
+  grid-area: 2/3/4/3;
+}
+.grid-items5 {
```
Comment (naumanzchaudhry) [inline my-gallary/styles.css]: Why are these classes empty. If they are not needed, should be removed

File: my-gallary/styles.css [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+}
+.grid-items6 {
+}
+footer {
+  margin-top: 20px;
+
+  font-size: 14px;
+  font-weight: 700;
+
+  color: #b9bdcf;
```
Comment (naumanzchaudhry) [inline my-gallary/styles.css]: Are we not using css variables?

**Review by naumanzchaudhry — CHANGES_REQUESTED** [review]
Please fix mentioned changes

**Comment by aliakhizer** [comment]
I delete it, it's not needed, I mentioned it just for my understanding...

**Comment by aliakhizer** [comment]
I have used 2 css custom  variables.. for color and border radius..

File: my-gallary/index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+          <div class="box1">
+            <h4>2,424</h4>
+            <p>Followers</p>
+          </div>
+          <div class="box1">
+            <h4>1,432</h4>
+            <p>Following</p>
+          </div>
+        </div>
+      </div>
```
Comment (hariscs) [inline my-gallary/index.html]: use semantic HTML, eg `header`

File: my-gallary/index.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      <div class="container">
+        <div class="profile-info">
+          <img src="images/profilePhoto.png" alt="not-found" />
+          <h2>Rodney cotton</h2>
+          <p>helisinki,FinLand</p>
+        </div>
+        <div class="profile-post">
+          <div class="box1">
+            <h4>100</h4>
+            <p>Posts</p>
```
Comment (hariscs) [inline my-gallary/index.html]: both of them should be `span` tags

File: my-gallary/styles.css [hunk 5]
```diff
@@ -0,0 +1,152 @@
+:root {
+  --gray-color: #a9a9a9;
+  --border-radius: 12px;
+}
+* {
+  margin: 0px;
+  padding: 0px;
+  font-family: "Montserrat", sans-serif;
```
Comment (hariscs) [inline my-gallary/styles.css]: use css variables for font-family

File: my-gallary/styles.css [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+
+.container-top {
+  width: 90%;
+  margin: 0 auto;
+}
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```
Comment (hariscs) [inline my-gallary/styles.css]: css class names should be meaningful

File: my-gallary/styles.css [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+  color: #4e5150;
+  margin-top: 25px;
+}
+
+.profile-info p {
+  color: var(--gray-color);
+  font-size: 14px;
+  line-height: 12px;
+  margin-top: 4px;
+}
```
Comment (hariscs) [inline my-gallary/styles.css]: add classes to these elements istead od targeting the element direclty

File: my-gallary/styles.css [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+
+.profile-post {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 10%;
+  margin-top: 10px;
+}
+.box1 {
```
Comment (hariscs) [inline my-gallary/styles.css]: don't use numbers in class names

File: my-gallary/styles.css [hunk 9]
```diff
@@ -0,0 +1,152 @@
...
+  margin-top: 30px;
+}
+
+.gallary-container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  /* grid-template-rows: 109px 109px 109px; */
+  gap: 10px;
+
+  /* max-width: 280px; */
```
Comment (hariscs) [inline my-gallary/styles.css]: remove unused code

**Review by hariscs — CHANGES_REQUESTED** [review]

**Review by hariscs — APPROVED** [review]

**Pull request merged**